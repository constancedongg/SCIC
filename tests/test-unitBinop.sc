float '{cm} x;
float '{m} y;

int func main(){
    y = 1.022;
    x = 100.9;
    float '{m} z = x + y;
    printf(z); 
    float '{m} large = 1.1;
    float '{cm} small = 110.0;
    printb(large == small);
    return 0;
}