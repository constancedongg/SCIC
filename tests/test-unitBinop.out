2.031
1